test/recipes/66-test_ossl_store.t

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env perl
+# Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use strict;
+use warnings;
+
+use OpenSSL::Test::Simple;
+use OpenSSL::Test qw/:DEFAULT srctop_file/;
+
+setup("test_ossl_store");
+
+plan tests => 1;
+
+ok(run(test(["ossl_store_test", "-in", srctop_file("test","testrsa.pem")])));
```

Comment:
nit: missing space after comma

Reply:
At some point a code style checker might be a good idea.

Reply:
Fixed.